src/TestTask/components/Amount/Amount.tsx

```diff
@@ -0,0 +1,253 @@
+import React, { useRef, CSSProperties } from 'react'
```

Comment:
next component, that already exists

Reply:
Noted. Please see my response to this comment: https://github.com/invariant-labs/webapp/pull/297#discussion_r882119874